debs/SPECS/4.2.0/wazuh-manager/debian/changelog

```diff
@@ -4,6 +4,36 @@ wazuh-manager (4.2.0-RELEASE) stable; urgency=low
 
  -- Wazuh, Inc <info@wazuh.com>  Mon, 26 Apr 2021 11:51:55 +0000
 
+wazuh-agent (4.1.5-RELEASE) stable; urgency=low
+
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Thu, 22 Apr 2021 16:50:05 +0000
+
+wazuh-agent (4.1.4-RELEASE) stable; urgency=low
+
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Mon, 29 Mar 2021 16:23:09 +0000
+
+wazuh-agent (4.1.3-RELEASE) stable; urgency=low
+
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Sat, 20 Mar 2021 13:41:26 +0000
+
+wazuh-agent (4.1.2-RELEASE) stable; urgency=low
+
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Mon, 08 Mar 2021 14:00:25 +0000
+
+wazuh-agent (4.1.1-RELEASE) stable; urgency=low
+
+ * More info: https://documentation.wazuh.com/current/release-notes/
+
+-- Wazuh, Inc <info@wazuh.com>  Fri, 05 Mar 2021 13:24:41 +0000
+
```

Comment:
Add 3.13.3 to the versioning, thanks.

Reply:
Fixed here: https://github.com/wazuh/wazuh-packages/pull/761/commits/e4f1ac9588d3fa88e9c9241466a33ba4cc0284c5 and https://github.com/wazuh/wazuh-packages/pull/761/commits/8dbb9e320792dac67bb944d8e27e9a3706724fa0